trial-config-service/src/test/resources/collections/Trial_Config_Service.postman_collection.json

```diff
@@ -0,0 +1,381 @@
+{
+	"info": {
```

Comment:
how was this file generated? can we add to the readme how to update this

Reply:
file has been removed as a new testing approach has been agreed.